pkg/operator/vitesstopo/prune_cells.go

```diff
@@ -80,7 +80,8 @@ func DeleteCells(ctx context.Context, ts *topo.Server, recorder record.EventReco
 	resultBuilder := &results.Builder{}
 
 	for _, cellName := range cellNames {
-		if err := ts.DeleteCellInfo(ctx, cellName); err != nil {
+		// topo.NoNode is the error type returned if we can't find the cell when deleting. This ensures that this operation is idempotent.
```

Comment:
I think we should avoid logging that we deleted it if we actually did nothing.

Reply:
Ok, pushed up a new commit to make it so we don't log in this case. I think by definition though that may make this non-idempotent, because don't the same thing multiple times results in different side effects (the first time logging an event, and then not logging every again). I'm not a purist though and may be wrong. The end outcome is idempotent enough I think regardless.